hw/formal/formal_conn.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Script to run connectivity test using Jasper Gold
+#
+# Usage: To run connectivity test on system level modules, type
+#   formal_conn -f top_earlgrey.csv
+#
+# More options:
+# -top: which top_level module to run
+# -p:  provide core file path
+# -gui: run with GUI
+# -cov: run coverage
+# -f: path to the csv file
+# -t: choose which formal tool to use, current only support jaspergold
+#
+# Example:
+#   formal_conn -gui -cov -f top_earlgrey_conn.csv
+#
+
+set -e
+
+unset CORE_PATH
+CORE_PATH=systems:top_earlgrey
```

Comment:
Just setting it at the top like this is much more sensible than what I suggested! As a nit: you no longer need the `unset` in this case (so you might want to lose it if you're re-pushing anyway)

Reply:
Thank you Rupert for your suggestions and patience..Sorry I am pretty new to batch script. Hopefully i got it this time.. I am going to merge the PR, but if somehow I still misunderstand your suggestion, please do not hesitate to correct me! I. will open a new PR to fix it.

Reply:
Looks great, thanks!